common/lib/xmodule/xmodule/modulestore/inheritance.py

```diff
@@ -5,10 +5,10 @@
 
 from django.conf import settings
 
-from xmodule.partitions.partitions import UserPartition
 from xblock.fields import Scope, Boolean, String, Float, XBlockMixin, Dict, Integer, List
 from xblock.runtime import KeyValueStore, KvsFieldData
 from xmodule.fields import Date, Timedelta
+from xmodule.partitions.partitions import UserPartition
```

Comment:
I can move this import back to its original location so this file isn't part of the change set.